### PR TITLE
Enterprise: add dependencies for upcoming features

### PR DESCRIPTION
**Pull request by markelog**
See enterprise issues

### CODE REVIEW

**Review by torkelo — APPROVED** [review]

**Review by torkelo — CHANGES_REQUESTED** [review]
go.mod and the vendored dependencies should also be checked in

**Comment by xlson** [comment]
As Torkel mentioned, the dependencies (in the `vendor` folder) and `go.mod` changes should be checked in. You achieve this by following this guide: https://github.com/grafana/grafana/blob/master/UPGRADING_DEPENDENCIES.md#go-dependencies

1. `go get ...`
2. `go mod tidy`
3. `go mod vendor`

**Comment by markelog** [comment]
Updated, waiting for the CI to pass and merging after that

File: pkg/extensions/main.go [hunk 1]
```diff
@@ -1,10 +1,15 @@
 package extensions
 
 import (
+	// Need to remove "github.com/robfig/cron" with v3
```
Comment (xlson) [inline pkg/extensions/main.go]: Is there plans for using the cron dependency outside of enterprise? Or do you mean something else?

**Review by xlson — APPROVED** [review]

File: pkg/extensions/main.go (hunk 1 above)
Comment (markelog) [reply pkg/extensions/main.go]: The `ldapsync` uses cron v1 and new stuff using v3, this comment basically means that `ldapsync` should switch to newer version of the cron after we have the time, was planning to create an issue when this pr lands

Comment (xlson) [reply pkg/extensions/main.go]: okay. Suggestion for comment: // Move ldapsync from cron to cron.v3 and remove the cron (v1) dependency.

Comment (markelog) [reply pkg/extensions/main.go]: Done!

**Review by torkelo — APPROVED** [review]

**Pull request merged**